var/spack/repos/builtin/packages/hpe-mpi/package.py

```diff
@@ -20,7 +21,11 @@ class HpeMpi(Package):
 
     version('2.21',
             sha256='2f27ad2e92ef0004b9a4dfb3b76837d1b657c43ff89f4deef99be58a322a80b7',
-            url='file:///gpfs/bbp.cscs.ch/apps/hpc/download/hpe-mpi/hpe-mpi-2.21.tar.xz')
+            url='file:///gpfs/bbp.cscs.ch/apps/hpc/download/hpe-mpi/hpe-mpi-2.21.tar.xz',
+            preferred=True)
+    version('2.21.hmpt',
```

Comment:
So the new deployment should use this one?

Asking for the next merge…

Reply:
yes and we will verify if everything works as before (as promised by HPE; in case of issues, we might need to revert)